custom/ilsgateway/models.py

```diff
@@ -190,17 +190,14 @@ class ReportingModel(models.Model):
     external_id = models.PositiveIntegerField(db_index=True, null=True)
 
     class Meta(object):
-        app_label = 'ilsgateway'
+        abstract = True
```

Comment:
This seems scary. Are you sure which one won out? (guessing hte second one)

I think we should probably include `app_label` (it's included elsewhere) and run `makemigrations` to make sure there aren't any changes

Reply:
I inspected it in a shell and ```ReportingModel._meta.app_label``` evaluates to ```'ilsgateway'``` whether or not the ```app_label``` is explicitly defined